setup.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Here as well as in all other Python files add a copyright line to attribute the authorship to yourself. See https://github.com/kiwitcms/junit-plugin/pull/10